cmd/client_changefeed.go

```diff
@@ -143,6 +145,9 @@ func newQueryChangefeedCommand() *cobra.Command {
 				taskStatus = append(taskStatus, captureTaskStatus{CaptureID: captureID, TaskStatus: status})
 			}
 			meta := &cfMeta{Info: info, Status: status, Count: count, TaskStatus: taskStatus}
+			if info == nil {
+				log.Warn("this changefeed has been deleted, the residual meta data will be completely deleted within 24 hours.")
```

Comment:
How do we test it if it is get deleted after 24 hours?

Reply:
i will add a unit test for it

Reply:
PTAL again